analyses/collapse-rnaseq/00-create-rsem-files.R

```diff
@@ -0,0 +1,78 @@
+# Author: Komal S. Rathi
+# Date: 11/09/2019
+# Function: 
+# merges all RSEM files into two RDS objects corresponding to polya and stranded data
```

Comment:
Am I correct in understanding that this script is run during data preparation? The input files for this are not part of the data download, but this is the script used to create the data download files?

Reply:
Yes - you are right. The ones before (fpkm/counts for polya and stranded) were incorrect so I wrote a script to regenerate those.

Reply:
These updated files will be incorporated into the data release then (or already have been?)

Reply:
They have been incorporated in V9.